CrackShot/weapons/firelaser.yml

```diff
@@ -0,0 +1,16 @@
+firelaser:  
+      Item_Information:  
+          Item_Name: "&6F&ei&6re_&6L&ease&6r"  
+          Item_Type: 417  
+      Shooting:  
+          Cancel_Left_Click_Block_Damage: true  
+          Cancel_Right_Click_Interactions: true  
+          Projectile_Amount: 1  
+          Projectile_Type: energy  
+          Projectile_Subtype: 40-1-NONE-0  
+          Projectile_Damage: 12  
+          Delay_Between_Shots: 10  
+      Reload:  
+          Enable: false  
+      Hit_Events:  
+          Enable:   
```

Comment:
this doesn't look right, there's no value after enable

idk if it's supposed to be

Hit_Events: Enable

or

Hit_Events:
    Enable: true

Check with another hero